.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -0,0 +1,43 @@
+---
+name: ':rocket: Feature Request'
+about: Suggest a new idea/feature for improving the game
+labels: 'c.Feature :rocket:'
```

Comment:
Can you add `title: ''` and `assignees: ''` and rename the label to be consistent with the bug template?